migration/iD10/oracle/202209141520_IDEMPIERE-5396.sql

```diff
@@ -0,0 +1,28 @@
+-- IDEMPIERE-5396 Replace CreateFrom form with Info Window Process
```

Comment:
why do we have scripts in iD10 - if this is intended for i9?

Reply:
we have to split the create view script for i9 and i10 due the incompatibility between r9 and r10 introduce by IDEMPIERE-5232